t/129-ssl-socket.t

```diff
@@ -2508,6 +2545,41 @@ SSL reused session
         #set $port 5000;
         set $port $TEST_NGINX_MEMCACHED_PORT;
 
+        content_by_lua_block {
+            local sock = ngx.socket.tcp()
+            sock:settimeout(7000)
+
+            local ok, err = sock:connect("openresty.org", 443)
+            if not ok then
+                ngx.say("failed to connect: ", err)
+                return
+            end
+
+            ngx.say("connected: ", ok)
+
+            local session, err = sock:sslhandshake(1, 2, 3, 4, 5, 6)
```

Comment:
This test case assumes that the number of arg check is performed before arg type check, which assumes too much about the underlying implementation.